_includes/tutorials/message-ordering/kafka/code/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.21.0"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "1.1.1"
+    id "idea"
+    id "eclipse"
+}
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+version = "0.0.1"
+
+repositories {
+    jcenter()
+
+    maven {
+        url "https://packages.confluent.io/maven"
+    }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+    implementation "org.apache.avro:avro:1.10.0"
+    implementation "org.slf4j:slf4j-simple:1.7.30"
+    implementation "org.apache.kafka:kafka-streams:2.6.0"
```

Comment:
Perhaps use `kafka-clients` instead of all these Kafka Streams dependencies, since this KT doesn't use Streams?

Reply:
@mikebin I did not replace this line in https://github.com/confluentinc/kafka-tutorials/pull/587/commits/12e0b49c39a1035b172e96a1c1835347b5900090 because when I did, it resulted in 

```
java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/JsonNode
	at org.apache.kafka.clients.producer.internals.TransactionManager.bumpIdempotentEpochAndResetIdIfNeeded(TransactionManager.java:585)
	at org.apache.kafka.clients.producer.internals.Sender.runOnce(Sender.java:311)
	at org.apache.kafka.clients.producer.internals.Sender.run(Sender.java:240)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.JsonNode
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 4 more
```

@gAmUssA had a message on this in Slack, and sounds like it was fixed after 2.6.0 was released.  So I will leave it as-is for now.  I do not want to downgrade to 2.5.0 because it results in different behavior that relevant for this KT:

```
[main] INFO org.apache.kafka.clients.producer.KafkaProducer - [Producer clientId=myApp] Overriding the default retries config to the recommended value of 2147483647 since the idempotent producer is enabled.
```

Reply:
https://issues.apache.org/jira/browse/KAFKA-10378

Reply:
@mikebin thanks!  @gAmUssA thanks for the workaround.  https://github.com/confluentinc/kafka-tutorials/pull/587/commits/f03c5e4dc979d99bed65cc0477d45ecff9e99b72 should address this now